Content.Shared/Roles/SharedRoleSystem.cs

```diff
@@ -620,6 +625,30 @@ public void MindPlaySound(EntityUid mindId, SoundSpecifier? sound, MindComponent
 
         return antag.Requirements;
     }
+
+    /// <summary>
+    /// Returns the localized name of a role type's subtype and/or role type.
+    /// </summary>
+    /// <param name="roleType">The locale string of the role type</param>
+    /// <param name="subtype">The locale string of the subtype</param>
```

Comment:
These two can be made to require a `LocId` as input instead of a string to avoid implicit conversions. All current usages of this method already pass in a LocId.

Reply:
At one point it was all changed to string, and I forgot to change these back